cds_migrator_kit/videos/weblecture_migration/transform/models/video_lecture.py

```diff
@@ -0,0 +1,159 @@
+
+from .base import model as base_model
+
+
+class VideoLecture(CdsOverdo):
+    """Translation Index for CERN Video Lectures."""
+
+    __query__ = '8567_.x:"Absolute master path" 8567_.d:/mnt/master_share* -980__.C:MIGRATED -980__.c:DELETED -5831_.a:digitized'
+
+    __ignore_keys__ = {
+        "110__a",  # corporate author
+        "8567_y",  # File description
+        "111__c",  # Video location (indico location)
+        "518__r",  # Video/meeting location
+        "340__a",  # Physical medium -> curation field
+        "340__d",  # Physical medium/recording technique -> curation field
+        "961__x",  # Creation Date TODO? check with JY
+        "961__c",  # modification Date TODO? check with JY
+        "961__h",  # Hour? TODO? check with JY
+        "961__l",  # Library? TODO? check with JY
+        "964__a",  # Item owner TODO? check with JY
+        "916__d",  # Status week? TODO? check with JY
+        "901__u",  # Affiliation at Conversion? TODO? check with JY
+        "583__a",  # Action note / curation TODO? check with JY
+        "583__c",  # Action note / curation TODO? check with JY
+        "583__z",  # Action note / curation TODO? check with JY
+        "65027a",  # TODO Subject category = Event?
+        "111__a",  # Title (indico)
+        "111__9",  # Start date (indico)
+        "111__g",  # Event id (indico)
+        "111__z",  # End date (indico)
+        "084__a",  # Indico?
+        "084__2",  # Indico?
+        "8567_2",  # File system? 'MediaArchive'
+        "980__b",  # Secondary collection indicator
+        "542__d",  # Copyright holder
+        "542__g",  # Copyright date
+        "490__a",  # TODO Series
+        "8567_u",  # File url
+        "962__n",  # `Presented at` note (conference/linked document)
+        "962__b",  # `Presented at` record (conference/linked document)
+        "518__g",  # Lectures: conference identification
+        "490__v",  # Series: volume
+        "269__b",  # Name of publ.
+        "088__9",  # Report number (make it alternative identifier with cds reference?)
+        "088__z",  # Report number (make it alternative identifier with cds reference?)
+        # Files
+        "8564_q",  # File type (digitized) # TODO this record has both lecturemedia and DM https://cds.cern.ch/record/589875
+        "852__c",  # Location (Shelving/Library)
+        "852__h",  # Location (Shelving) example: https://cds.cern.ch/record/254588/
+        "852__a",  # Location (Shelving) example: https://cds.cern.ch/record/558348
+        "852__x",  # Location (Shelving/ type? DVD) example: https://cds.cern.ch/record/690000/
+        # IGNORE
+        "518__h",  # Lectures: Starting time
+        "300__2",  # Imprint
+        "300__b",  # Imprint
+        "300__a",  # Number of pages / duration
+        "250__a",  # Edition
+        "700__0",  # Author id (eg: AUTHOR|(CDS)2067852)
+        "518__l",  # Lectures: length of speech
+        # TODO copied from ssn
```

Comment:
I think you shouldn't copy from ssn by default but analyse your collection separately